### PR TITLE
Blog Post on using GitHub

### PR DESCRIPTION
**Pull request by kathweaver**

### CODE REVIEW

File: _posts/2016-02-11-GitHub.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+categories: [blog]  
+date: 2016-02-16 00:00:00 -0800  
+author: kathweaver  
+download: true  
+---
+
+For those new to GitHub, and I am one, it's very well documented.  First a quick overview of the process.
+
+Open Live Writer has two different repositories (a repository is just a place to store things).  One for the program itself
+and a second one for the documentation.  If you are reading this, you are reading documentation.
```
Comment (ScottIsAFool) [inline _posts/2016-02-11-GitHub.md]: Is there a reason you're forcibly splitting sentences? This should all just be one line, right?

File: _posts/2016-02-11-GitHub.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+The code is at [https://github.com/OpenLiveWriter](https://github.com/OpenLiveWriter) and the documentation is at 
+[https://github.com/OpenLiveWriter/OpenLiveWriter.Github.io](https://github.com/OpenLiveWriter/OpenLiveWriter.Github.io)
+
+The two best places to learn about Github, is Github itself.  Help is at [https://help.github.com/](https://help.github.com/)
+and [https://www.codecademy.com/](https://www.codecademy.com/) There is a walkthrough there.
+
+Unfortunately if you are adding to documentation or code, you do have to use both GitHub desktop and Git Shell,
+again the process for both are very well documented, with a tutorial loaded into Github Desktop.
```
Comment (ScottIsAFool) [inline _posts/2016-02-11-GitHub.md]: Same here

Comment (kathweaver) [inline _posts/2016-02-11-GitHub.md]: User issue

**Comment by kathweaver** [comment]
User issue <vbg> not used to markdown yet   -->  I think I fixed.

**Comment by ScottIsAFool** [comment]
You definitely want to have a look at http://markdownpad.com/ :)

**Comment by kathweaver** [comment]
Yeah, but I like Visual Studio code.  And yes, I hand code my html.

**Comment by kathweaver** [comment]
Thanks for merging!

**Pull request merged**